### ISSUE TITLE
Feature: Activate user by moderator or admin

### ISSUE THREAD

**Issue by sheepsy90** [nitely/Spirit#111]
In order to not allow everybody just registering in the forum and start doing things it would be great to have a mechanism that still confirms the email address but lets the user not compose entries as long as they where not actively confirmed as a 'member' by a moderator or admin.

**Comment by nitely** [comment]
> lets the user not compose entries as long as they where not actively confirmed as a 'member' by a moderator or admin.

Would be way better not to let them sign-in, IMHO. It'd be great for super private communities. Something like showing them a "your account is pending for approval" message and send them an email when it gets approved. Any reason why you would prefer the other way?

**Comment by sheepsy90** [comment]
I totally agree that a setting flag that switches between the current mode and your proposal would be totally enough. The consideration that I just would put in that the email confirmation would be nice in terms of making sure that they can access their account if they forgot their password.

**Comment by sheepsy90** [comment]
Shall I try to implement something ?

**Comment by nitely** [comment]
If you want this ASAP, then yeah, go ahead. I just created a [gitter room for Spirit](https://gitter.im/nitely/Spirit) so let me know there or at the Spirit's community if you hit a wall or are unsure about something.
